test_runner.sh

```diff
@@ -1,2 +1,7 @@
 #!/usr/bin/env bash
-which sudo &> /dev/null && sudo $@ || $@
+
+if command -v sudo &> /dev/null; then
+    sudo $@
+else
+    $@
+fi
```

Comment:
What does this script do?

Reply:
It runs the tests executables under sudo if it exists.

Reply:
Could you elaborate? Does it depend on different runners, like mac vs linux?

Reply:
I think the intention here is that when you run on a platform which has sudo, then use it but otherwise assume you are already a root user. To be honest I didn't investigate that much - it's in the MR 23 on the original libtelio repo and there are no information in that MR that would explain why it was done this way.

Reply:
Thanks for explaining.